.gitignore

```diff
@@ -4,4 +4,6 @@ spec/fixtures
 .gitconfig
 .bundle
 Gemfile.lock
-.idea
\ No newline at end of file
+.idea
+*.swp
```

Comment:
These `gitignore` entries are a good fit for your global `~/.gitignore`, but please don't add them here.